package-lock.json

```diff
@@ -4241,26 +4241,30 @@
       "version": "1.2.3",
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.3.tgz",
       "integrity": "sha512-X+57O5YkDTiEQGiw8i7wYc2nQgweIekqkepI8Q3y4wVlurgBt2SuwxTeYUYMZIGpLZH3r/TsMjczCMXE5ZOt7Q==",
+      "optional": true,
```

Comment:
Usually when we edit `package.json` manually we regenerate `package-lock.json` and commit both files.
Not sure what are the changes in `package-lock.json` and do we need them?

Reply:
it is removed in a subsequent commit. I can remove completely if you want